src/game/server/entities/foot/foot_pickup.h

```diff
@@ -0,0 +1,31 @@
+#ifndef GAME_SERVER_ENTITIES_FOOT_FOOT_PICKUP_H
+#define GAME_SERVER_ENTITIES_FOOT_FOOT_PICKUP_H
+
+#include <game/server/entity.h>
+
+class CFootPickup : public CEntity
+{
+public:
+	static constexpr int gs_PickupPhysSize = 14;
+
+	CFootPickup(CGameWorld *pGameWorld, int Type, int SubType = 0, int Layer = 0, int Number = 0);
+
+	void Reset() override;
+	void Tick() override;
+	void TickPaused() override;
+	void Snap(int SnappingClient) override;
+
+	int Type() const { return m_Type; }
+	int Subtype() const { return m_Subtype; }
+
+private:
+	int m_Type;
+	int m_Subtype;
+	int m_SpawnTickTeam[NUM_DDRACE_TEAMS];
```

Comment:
missing a prefix for array

```suggestion
	int m_aSpawnTickTeam[NUM_DDRACE_TEAMS];
```